src/rust/cryptography-x509/src/common.rs

```diff
@@ -327,6 +340,13 @@ pub struct RsaPssParameters<'a> {
     pub _trailer_field: u8,
 }
 
+#[derive(asn1::Asn1Read, asn1::Asn1Write, Hash, Clone, PartialEq, Eq, Debug)]
+pub struct DssParms<'a> {
```

Comment:
A part of me is tempted to leave this typo for the future parmesan joke, but we should probably make it say `DssParams`

Reply:
🤯 We should probably match the RFC so I guess let's not change it. Maybe a comment that links to that so I don't have this conversation with you every 6 months 😆

Reply:
Already changed the name and added a comment.